src/index.js

```diff
@@ -78,8 +78,16 @@ class Store {
         handler(payload)
       })
     })
-    if (!options || !options.silent) {
-      this._subscribers.forEach(sub => sub(mutation, this.state))
+    this._subscribers.forEach(sub => sub(mutation, this.state))
```

Comment:
I think we should not only provide deprecation message but also retain the silent flag's behavior until next major release.